ga4gh/cli.py

```diff
@@ -295,7 +306,7 @@ def addVariantSearchOptions(parser):
 
 def addVariantSetIdsArgument(parser):
     parser.add_argument(
-        "--variantSetIds", "-v",
+        "--variantSetIds", "-V",
```

Comment:
I know this is just a one-character change, but I think because of what I mentioned in the above comment it might be worth breaking this change out into a separate PR (there also might be broken tests associated with this change)